conjur-oss/templates/secrets.yaml

```diff
@@ -38,6 +38,29 @@ type: Opaque
 data:
   key: {{ .Values.dataKey | b64enc }}
 ---
+{{ if .Values.database.remotePassword }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ .Release.Name }}-database-remote-password
+  labels:
+    app: {{ template "conjur-oss.name" . }}
+    chart: {{ template "conjur-oss.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    app.kubernetes.io/name: {{ .Release.Name }}
+    app.kubernetes.io/component: service
+{{- with .Values.conjurLabels }}
+{{ toYaml . | indent 4 }}
+{{- end }}
+  annotations:
+    helm.sh/hook: pre-install
+    helm.sh/hook-delete-policy: before-hook-creation
+type: Opaque
+data:
+  key: {{ .Values.database.remotePassword | b64enc }}
```

Comment:
The data here is only for remote password, we are expecting user to create the entire URL by himself?
Do we have guidelines of what we do for internal DB like:
`postgres://postgres:%s@%v-postgres/postgres?sslmode=require`
sslmode? or other Auroa connection string functionality it might supply